src/main/java/Duke.java

```diff
@@ -1,10 +1,180 @@
+    }
+
+    public static void printIndentationAndDivider() {
+        System.out.print(INDENTATION);
+        System.out.println(DIVIDER);
+    }
+
+    public static void printWordsWithIndentation(String words) {
+        System.out.print(INDENTATION);
+        System.out.println(words);
+    }
+
+    private static void printStartingMessage() {
+        System.out.println("Hello from\n" + LOGO);
+        printIndentationAndDivider();
+        printWordsWithIndentation(HELLO_MESSAGE_2);
+        printWordsWithIndentation(HELLO_MESSAGE_3);
+        printIndentationAndDivider();
+        System.out.println();
+    }
+
+    private static void printGoodbyeMessage() {
+        printIndentationAndDivider();
+        printWordsWithIndentation(GOODBYE_MESSAGE);
+        printIndentationAndDivider();
+    }
+
+    private static void processInput(String input) {
+        switch(input.split(" ")[0].toLowerCase()) {
+        case "list" :
+            executeListCase();
+            break;
+        case "done" :
+            executeDoneCase(input);
+            break;
+        case "todo" :
+            executeTaskCase(input, TODO_STARTING_INDEX,TaskType.TODO);
+            break;
+        case "deadline" :
+            executeTaskCase(input, DEADLINE_STARTING_INDEX,TaskType.DEADLINE);
+            break;
+        case "event" :
+            executeTaskCase(input, EVENT_STARTING_INDEX,TaskType.EVENT);
+            break;
+        default :
+            printIncorrectInputMessage();
+        }
+    }
+
+    private static void printTaskMessage() {
+        printIndentationAndDivider();
+        printWordsWithIndentation(ADDED_TO_LIST);
+        printWordsWithIndentation(tasks[Task.getTotalTasks() - 1].getStatusIconAndDescription());
+        printWordsWithIndentation(notifyNumberOfTasks());
+        printIndentationAndDivider();
+        System.out.println();
+    }
+
+    private static void printIncorrectInputMessage() {
+        printIndentationAndDivider();
+        printWordsWithIndentation(TYPE_SUITABLE_COMMAND_MESSAGE);
+        printIndentationAndDivider();
+        System.out.println();
+    }
+
+    private static void executeTaskCase(String input, int starting_index, TaskType type) {
+        String description = input.strip().substring(starting_index);
+        boolean isFine = addTask(description, type);
+        if(isFine) {
+            printTaskMessage();
+        }else {
+            printIncorrectInputMessage();
+        }
+    }
+
+    private static String[] parseInputForDifferentTask(String input,TaskType type) {
+        String[] parsedOutput = {};
+        switch(type) {
+        case TODO:
+            parsedOutput = new String[]{input};
+            break;
+
+        case DEADLINE:
+            parsedOutput = input.split(DEADLINE_DESCRIPTION_AND_DATE_SPLITTER);
+            break;
+
+        case EVENT:
+            parsedOutput = input.split(EVENT_DESCRIPTION_AND_DATE_TIME_SPLITTER);
+            break;
+        }
```

Comment:
Perhaps you could add a `default` branch? The `default` branch can be used to raise a suitable error as I think the execution should not reach the `default` branch. I also noticed this same issue in your other `switch` statements.